packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts

```diff
@@ -442,3 +449,11 @@ async function _buildProject(opts: ProjectOptions<TransformerFactoryArgs>) {
   });
   return transform.transform(userProjectConfig.schema.toString());
 }
+
+async function _updateCurrentMeta(opts: ProjectOptions<TransformerFactoryArgs>) {
```

Comment:
nit: this looks like a helper util we could store in utils
https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-provider-awscloudformation/src/graphql-transformer/utils.ts